ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1602,7 +1602,7 @@ tests:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
 - as: azure-ipi-private-workers-marketimage-p2-f28
-  cron: '@yearly'
+  cron: 30 23 1 4 *
```

Comment:
The `yearly` was done in https://github.com/openshift/release/pull/39542 purposely by @jinyunma

Reply:
Yeah, they are on purpose. The owner of the azure account is asking us to pend the testing on the account to catch up the delayed payment.

Reply:
My PR does not work differently, still once every year.

Reply:
Ah, got it.

Reply:
I'm fine with the changes. I'll let @jinyunma to decide

Reply:
it's also okay for me.